.github/workflows/linux-ci-helper.sh

```diff
@@ -53,9 +53,9 @@ fi
 #-----------------------------------------------------------
 CONTAINER_FULLNAME=$1
 # shellcheck disable=SC2034
-CONTAINER_OSNAME=$(echo "${CONTAINER_FULLNAME}" | sed 's/:/ /g' | awk '{print $1}')
+CONTAINER_OSNAME=$(echo "${CONTAINER_FULLNAME}" | cut -d: -f1)
 # shellcheck disable=SC2034
-CONTAINER_OSVERSION=$(echo "${CONTAINER_FULLNAME}" | sed 's/:/ /g' | awk '{print $2}')
+CONTAINER_OSVERSION=$(echo "${CONTAINER_FULLNAME}" | cut -d: -f2)
```

Comment:
Fedora 42 doesn't have `awk` at this point in the script and it is installed later.

Reply:
Thank you, no problem.